.gitignore

```diff
@@ -1,3 +1,3 @@
 /vendor/
 /composer.phar
-/composer.lock
```

Comment:
Please keep this line and remove `composer.lock` from the pull request.

Reply:
I can remove it but why you don't want to have `composer.lock` file in repository? Thanks to `composer.lock` everyone who will be working on that library will have the same version of dependencies.

Reply:
Because the `composer.lock` file is not ment do be committed to the repository for libraries.

It also causes Travis to fail, because Travis tests this library with various PHP versions but the `composer.lock` pins the dependencies to specific versions that are not compatible with older PHP versions.

Reply:
ok, In such case I will remove composer.lock file